packages/wmr/src/plugins/wmr/client.js

```diff
@@ -2,6 +2,19 @@ function log(...args) {
 	console.info('[wmr] ', ...args);
 }
 
+/**
+ * Strip ansi colors from a string. Inlined from kolorist:
+ * https://github.com/marvinhagemeister/kolorist/blob/87a6402ec3a99ee4714ffb3f9bb54fd1d16174bd/src/index.ts#L62-L66
+ * @param {string} str
+ * @returns {string}
+ */
+function stripColors(str) {
```

Comment:
Would it make sense to invoke this on the server instead? It seems like it could be done [here](https://github.com/preactjs/wmr/pull/520/files#diff-618ca7c6f916be9a66704d923d39b5b0c2ef91ae8636e84409ec3ade9fad0561R94).

Reply:
Good call, updated the PR 👍